src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.DefaultItems.targets

```diff
@@ -186,6 +228,14 @@ Copyright (c) .NET Foundation. All rights reserved.
     
   </Target>
 
+  <Target Name="FixAspNetReference" AfterTargets="ResolveFrameworkReferences">
+    <ItemGroup>
+      <PackageReference>
+        <Publish Condition="'%(Identity)' == 'Microsoft.AspNetCore.App'">true</Publish>
```

Comment:
Shouldn't it come out out ResolveFrameworkReferences this way?

Reply:
Turns out it does, so this was unnecessary.  I think this was a leftover change from previous work I was doing.